cosmic_ray/test/test_operators.py

```diff
@@ -86,7 +86,7 @@ def test_activation_record_created(operator, code):
 @pytest.mark.parametrize('operator,code', OPERATOR_SAMPLES)
 def test_no_activation_record_created(operator, code):
     node = ast.parse(code)
-    core = MutatingCore(1)
+    core = MutatingCore(-1)
```

Comment:
Yes. In this test you are looking for a mutation/node which doesn't exist but value of 1 now hits a valid mutation.